maintenance_plan/views/maintenance.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+
+    <!-- maintenance.kind : views -->
+    <record id="maintenance_kind_view_tree" model="ir.ui.view">
+        <field name="name">maintenance.kind.tree</field>
+        <field name="model">maintenance.kind</field>
+        <field name="arch" type="xml">
+            <tree string="Maintenance Kinds" editable="true">
```

Comment:
Runbot fails with following error:

```
  File "/home/odoo/OCB-11.0/odoo/addons/base/ir/ir_ui_view.py", line 350, in _check_xml
    raise ValidationError(_('Invalid view definition'))
odoo.tools.convert.ParseError: "Invalid view definition
None" while parsing /home/odoo/build/OCA/maintenance/maintenance_plan/views/maintenance.xml:6, near
<record id="maintenance_kind_view_tree" model="ir.ui.view">
        <field name="name">maintenance.kind.tree</field>
        <field name="model">maintenance.kind</field>
        <field name="arch" type="xml">
            <tree string="Maintenance Kinds" editable="true">
                <field name="name"/>
                <field name="active"/>
            </tree>
        </field>
    </record>
```

Probably is the `editable="true"` causing the error. Could you fix it?

Also please remove the deprecated `string="Maintenance Kinds"` (see https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#views)